.github/workflows/ci.yml

```
@@ -40,6 +40,7 @@ jobs:
     container:
```

```suggestion
    timeout-minutes: 20
    container:
```
Can we also add time-out here? Because it seems that the job stuck and default timeout is 360 minutes